src/puppetlabs/ezbake/core.clj

```diff
@@ -699,6 +707,38 @@ Additional uberjar dependencies:
 (defmulti action
   (fn [action & args] action))
 
+(defn munge-deps
```

Comment:
Since there's a million ways you can transform one structure into another in Clojure, I think a simple example input and example output in the docstring here would speak volumes.

You don't even need to show a real input and real output (which would be huge), just enough to show the structure before and after.